helm-chart/values.yaml

```diff
@@ -249,7 +249,6 @@ keydb:
   enabled: true
   external:
     host: 127.0.0.1
-  imageTag: x86_64_v6.3.4
```

Comment:
Should we specify x86_64_v6.3.2 explicitly? Even it is a default value, can it be changed by someone outside?

Reply:
> Should we specify x86_64_v6.3.2 explicitly? Even it is a default value, can it be changed by someone outside?

It could be changed in either case, since all values in this file are overridable.

> Does it support Mac deployment (ARM arch)?

I cannot verify this, not having a Mac myself, but I'm pretty sure that Docker on Mac uses VMs, so it can run either `x86_64` or `arm64` images.

But for the record, KeyDB has `arm64` images too.